crates/fluvio-hub-util/src/utils.rs

```diff
@@ -4,13 +4,13 @@ use std::io::copy;
 
 use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
-use surf::http::mime;
-use surf::StatusCode;
 use tracing::{debug, info};
 
 use fluvio_hub_protocol::{PackageMeta, Result, HubError};
 use fluvio_hub_protocol::constants::HUB_PACKAGE_EXT;
-
+use reqwest;
+use reqwest::header::{AUTHORIZATION, CONTENT_TYPE};
+use reqwest::StatusCode;
```

Comment:
Whoops, these should be moved to the block above. I think `rustfmt.toml` could be edited to add

```
group_imports = "StdExternalCrate"
imports_granularity = "Module"
```

as a more general solution here, but that's probably out of scope (and an extremely noisy change for me to make).